### PR TITLE
Adds License & CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jfmercer**
## Adds License & CONTRIBUTING.md
- Adds CONTRIBUTING.md
- Adds LICENSE

### CODE REVIEW

**Comment by gavD** [comment]
this is the public bit and should be licensed accordingly (BSD3 or whatever we were using). We should also have the David and NPM badges as per the others OS projects.

**Comment by jfmercer** [comment]
This PR includes the BSD-3 license. I will add the David & NPM badges in a separate PR.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Radify, Inc.
+All rights reserved.
```
Comment (gavD) [inline LICENSE:2]: is this BSD3?

Comment (jfmercer) [inline LICENSE:2]: Yes. Here is [the template](http://choosealicense.com/licenses/bsd-3-clause/). The first two lines are:

```
Copyright (c) [year], [fullname]
All rights reserved.
```

**Pull request merged**